src/settings.ts

```diff
@@ -74,57 +74,55 @@ export class Settings {
    * @return {string} The bearer token.
    */
   public static getBearerToken(): string {
-    return Settings.readFileContent("~/.cloco/config/cloco_token");
+    return Settings.readFileContent(`${process.env.HOME}/.cloco/config/cloco_token`);
```

Comment:
feel free to keep for a later TODO, but I've a suspicion this isn't going to x-platform - people can `npm install cloco-node-client` on any old O/S.  Had it been me I'd have made a local prop for `homeDir: string`, or maybe a `getHome(): string` and then used that throughout so it's nice and easy to change when the time comes, but I guess that could also be done when the time comes to worry about it.